data-raw/DATASET.R

```diff
@@ -1,22 +1,23 @@
-## code to prepare `DATASET` dataset goes here
+## code to prepare `MBdelayed` dataset goes here
+library(simtrial)
 library(tibble)
 set.seed(6671)
-ds <- simPWSurv(
+ds <- sim_pw_surv(
   n = 200,
-  enrollRates = tibble(rate = 200 / 12, duration = 12),
-  failRates = tribble(
-    ~Stratum, ~Period, ~Treatment, ~duration, ~rate,
+  enroll_rate = tibble(rate = 200 / 12, duration = 12),
+  fail_rate = tribble(
+    ~stratum, ~period, ~treatment, ~duration, ~rate,
     "All", 1, "Control", 42, log(2) / 15,
```

Comment:
Please try update "Control" to "control", and "Experimental" to "experimental".

Reply:
Thanks @LittleBeannie! That was indeed the problem. The call to `sim_pw_surv()` used the default value for the argument `block`, which now uses lowercase: `block = c(rep("control", 2), rep("experimental", 2))`